lib/demo_web/templates/user_reset_password/edit.html.eex

```diff
@@ -0,0 +1,26 @@
+<h1>Reset password</h1>
+
+<%= form_for @changeset, Routes.user_reset_password_path(@conn, :update, @token), fn f -> %>
```

Comment:
```suggestion
<%= form_for @changeset, Routes.user_reset_password_path(@conn, :update, @token), [method: "put"], fn f -> %>
```

Reply:
@aptinio this should be not necessary as the method is lifted from the `@changeset` state.

Reply:
Ah, right. It was a typo on my end... `form_for @conn` instead of `form_for @changeset`. :sweat_smile: 

Thanks @josevalim!